src/phongo_compat.h

```diff
@@ -70,12 +70,24 @@
 #define phongo_compat_object_handler_type zend_object
 #define PHONGO_COMPAT_GET_OBJ(val) val
 #define PHONGO_COMPAT_SET_COMPARE_OBJECTS_HANDLER(type) php_phongo_handler_##type.compare = php_phongo_##type##_compare_objects;
+#define phongo_compat_property_accessor_name_type zend_string
+#define PHONGO_COMPAT_PROPERTY_ACCESSOR_NAME_TO_STRING(value, key, len) \
+	do {                                                                \
+		(key) = ZSTR_VAL((value));                                      \
+		(len) = ZSTR_LEN((value));                                      \
```

Comment:
Why is there floating backslashes ate the end of the line?

Reply:
Macros in C are only single-line, so if you want to add a multi-line macro you need to escape the line break with a backslash. Our coding standard then enforces them to be aligned at the end of the longest line.

Reply:
So this is it. Thanks, I've just learned about this setting, which I've never needed before.